lib/fastlane/actions/xcode_server_get_assets.rb

```diff
@@ -0,0 +1,299 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      XCODE_SERVER_GET_ASSETS_PATH = :XCODE_SERVER_GET_ASSETS_PATH
+      XCODE_SERVER_GET_ASSETS_ARCHIVE_PATH = :XCODE_SERVER_GET_ASSETS_ARCHIVE_PATH
+    end
+
+    class XcodeServerGetAssetsAction < Action
+
+      require 'excon'
+      require 'json'
+      require 'fileutils'
+
+      def self.run(params)
+
+        host = params[:host]
+        bot_name = params[:bot_name]
+        integration_number_override = params[:integration_number]
+        target_folder = params[:target_folder]
```

Comment:
I like how once you define it, Sublime makes it super easy to use in multiple places below. But good to know, thanks! :blush: